charts/prometheus-mysql-exporter/templates/deployment.yaml

```diff
@@ -36,18 +36,23 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          args:
+          {{- if .Values.config.logLevel }}
```

Comment:
I was going to make a comment on the indentation  but it looks like it's all over the place on the file. But perhaps make all this "args" block have a consistent indentation  it's a good start

Reply:
Thanks @Juanchimienti for the review

I made a change on the collector args indentation in order to be more consistent on that file

Tell me if it's ok 😉